[client-api] Typescript error due to missing types include

**Describe the bug**
When checking types on my own project where I use storybook it runs an error indicating that a type of a library used by storybook is missing and treats it like "any"

**To Reproduce**
run tsc --noEmit in a project that uses storybook

**Expected behavior**
No errors

**Screenshots**
```
$ tsc --noEmit -p tsconfig.all.json
node_modules/@storybook/client-api/dist/queryparams.d.ts:1:16 - error TS7016: Could not find a declaration file for module 'qs'. '/hello-web-components/node_modules/qs/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/qs` if it exists or add a new declaration (.d.ts) file containing `declare module 'qs';`

1 import qs from 'qs';
                 ~~~~


Found 1 error.
```

**Additional context**
Latest storybook beta version

--- a/lib/client-api/package.json
+++ b/lib/client-api/package.json
@@ -34,6 +34,7 @@
     "@storybook/client-logger": "6.0.0-beta.26",
     "@storybook/core-events": "6.0.0-beta.26",
     "@storybook/csf": "0.0.1",
+    "@types/qs": "^6.9.0",
     "@types/webpack-env": "^1.15.2",
     "core-js": "^3.0.1",
     "global": "^4.3.2",

